### ISSUE TITLE
`@graphql-eslint/no-scalar-result-type-on-mutation` errors on both the input and return type

### ISSUE THREAD

**Issue by dmregister** [dotansimha/graphql-eslint#796]
**Describe the bug**
When `@graphql-eslint/no-scalar-result-type-on-mutation` is set to error, there are 2 eslint errors that display in the console:
1. input argument type
2. output argument type (correct)

```
[lint]   153:25  error  Unexpected scalar result type "String"   @graphql-eslint/no-scalar-result-type-on-mutation
[lint]   153:35  error  Unexpected scalar result type "Boolean"  @graphql-eslint/no-scalar-result-type-on-mutation
[lint]
[lint] ✖ 2 problems (2 errors, 0 warnings)
[lint]
```

Should this only be reporting the return type if they are scalars?

**To Reproduce**
```graphql
type Mutation {
  example(exampleInput: String!): Boolean!
}
```

enable the eslint rule:
```
'@graphql-eslint/no-scalar-result-type-on-mutation': 'error',
```

**Expected behavior**
This should only error on the return type of the mutation.

**Environment:**

- OS: MacOS Big Sur 11.6
- `@graphql-eslint/...`: 3.0.0
- NodeJS: v14.15.5

**Additional context**

**Comment by dotansimha** [comment]
Fixed in `@graphql-eslint/eslint-plugin@3.0.1`.
